tools/targets.py

```diff
@@ -497,6 +497,13 @@ def ncs36510_addfib(t_self, resources, elf, binf):
         from tools.add_fib import add_fib_at_start
         print("binf ", binf)
         add_fib_at_start(binf[:-4])
+
+class RTL8195ACode:
+    """RTL8195A Hooks"""
+    @staticmethod
+    def binary_hook(t_self, resources, elf, binf):
+        from tools.build_hooks.REALTEK_RTL8195AM import rtl8195a_elf2bin
```

Comment:
Since I asked you to move the location of the post-bulid hook, you will have to update this line to reflect that new location.